test/controller.go

```diff
@@ -134,7 +134,7 @@ func seedTestData(d Data) (Clients, Informers) {
 
 // GetTaskRunController returns an instance of the TaskRun controller/reconciler that has been seeded with
 // d, where d represents the state of the system (existing resources) needed for the test.
-func GetTaskRunController(d Data) (*controller.Impl, *observer.ObservedLogs, Clients) {
+func GetTaskRunController(d Data) (*controller.Impl, *observer.ObservedLogs, Clients, Informers) {
```

Comment:
nit: can be done in follow-ups, but 4 returns value is a bit of a smell :angel: :stuck_out_tongue_closed_eyes: (could be replace by a struct or sthg else) :angel:

Reply:
Yeaaaaaah, good point. I'll do that in a bit. =)

Reply:
and done. =)